src/app/shared/services/filter-data.service.ts

```diff
@@ -2,8 +2,6 @@ import { Injectable } from '@angular/core';
 
 @Injectable()
 export class FilterDataService {
```

Comment:
It should be now something like `GridService`since we are managing columns and filters both.

Reply:
changed